src/meta/processors/admin/Balancer.cpp

```diff
@@ -123,16 +123,20 @@ bool Balancer::getAllSpaces(std::vector<GraphSpaceID>& spaces, kvstore::ResultCo
     return true;
 }
 
-Status Balancer::buildBalancePlan() {
+cpp2::ErrorCode Balancer::buildBalancePlan(std::vector<HostAddr> hostDel) {
```

Comment:
genTask parameter is left reference, so maybe std::vector<HostAddr> &hostDel is better